tests/src/test/java/alluxio/shell/command/SetTtlCommandIntegrationTest.java

```diff
@@ -99,9 +99,27 @@ public void setTtlWithInvalidOperationValue() throws Exception {
   }
 
   @Test
-  public void setTtlNegative() throws Exception {
-    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 1);
-    mFsShell.run("setTtl", "/testFile", "\"-1\"");
-    Assert.assertTrue(mOutput.toString().contains("TTL value must be >= 0"));
+  public void setTtlWithDifferentUnitTime() throws Exception {
+    String filePath = "/testFile";
+    FileSystemTestUtils.createByteFile(mFileSystem, filePath, WriteType.MUST_CACHE, 1);
+    Assert.assertEquals(Constants.NO_TTL,
+        mFileSystem.getStatus(new AlluxioURI("/testFile")).getTtl());
+
+    AlluxioURI uri = new AlluxioURI("/testFile");
+    String[] timeUnits = {"", "ms", "millisecond", "s", "sec", "second", "m", "min", "minute", "h",
```

Comment:
it's hard to maintain these two lists, we need to manually make sure the corresponding items have the same index...

what about using a `Map<String, long>`?